pkg/kubelet/dockershim/docker_container.go

```diff
@@ -426,7 +426,10 @@ func (ds *dockerService) ContainerStatus(_ context.Context, req *runtimeapi.Cont
 	}
 
 	// Convert to unix timestamps.
-	ct, st, ft := createdAt.UnixNano(), startedAt.UnixNano(), finishedAt.UnixNano()
+	ct, st, ft := createdAt.UnixNano(), startedAt.UnixNano(), int64(0)
+	if state == runtimeapi.ContainerState_CONTAINER_EXITED {
```

Comment:
Adding reason here may help for code reading.

Reply:
updated